index.html

```diff
@@ -0,0 +1,258 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>Custom Cakes</title>
+    <link rel="stylesheet" type="text/css" href="Assets/Styles/main.css">
+</head>
+<body>
+    <header class="base-section header">
+        <nav class="header-nav">
+            <h1 class="header__title">
+                Cakes
+            </h1>
+            <div class="header-menu">
+                <div class="header-menu-btn">
+                    <img src="Assets/Styles/site/header/images/menu-burger.png" alt="menu" class="header-menu-btn__image" />
+                </div>
+                <div class="header-links">
+                    <a href="#" class="header-links__item">Наши торты</a>
+                    <a href="#" class="header-links__item">Индивидуальный заказ</a>
+                    <a href="#" class="header-links__item">Отзывы</a>
+                    <a href="#" class="header-links__item">О нас</a>
+                    <a href="#" class="header-links__item">Как заказать</a>
+                    <a href="#" class="header-links__item">Контакты</a>
+                </div>
+            </div>
+        </nav>
+        <div class="header__number">
+            +7 906 899 92 96
+        </div>
+    </header>
+    <main>
+        <section class="base-section main-section">
+            <div class="main-section-info">
+                <h1 class="main-section-info__title">
+                    Торты на заказ
+                </h1>
+                <h3 class="main-section-info__hand-made">
```

Comment:
please use more constructive names, as for me `main-section-info__subtitle` is more suitable